include/swift/AST/DiagnosticsSema.def

```diff
@@ -1080,6 +1080,8 @@ ERROR(cdecl_throws,none,
 
 ERROR(attr_methods_only,none,
       "only methods can be declared %0", (DeclAttribute))
+NOTE(access_control_in_protocol_detailed_info,none,
```

Comment:
You just seem to have removed the word `more`, which is not quite what @xwu asked.  Take a look at the naming scheme for notes in the rest of this file.